### DISCUSSION TITLE
Activate Other Data Source on Self-Hosted Docker

### DISCUSSION THREAD

**Discussion by rulim34** [ghostfolio/ghostfolio#2090] [Q&A]
Hi, how can I activate other data source such as Alpha Vantage, FMP, etc on the self-hosted instance using official docker image? What value should I pass to the DATA_SOURCES env?

**Comment by dtslvr** [comment]
Hi @rulim34

You can find the list of `DATA_SOURCES` [here](https://github.com/ghostfolio/ghostfolio/blob/main/prisma/schema.prisma#L267).

Please note: `COINGECKO`, `MANUAL` and `YAHOO` are officially supported in the self-hosted version at the moment. All other data sources are experimental.

**Reply by rulim34** [reply]
Yes, but I mean, how can I supply that value to the env var when I'm deploying to k8s using the official docker image? I tried with ```"['ALPHA_VANTAGE', 'MANUAL', 'YAHOO']"``` but it doesn't work.

**Reply by dtslvr** [reply]
I'm setting `DATA_SOURCES` like this in the `.env` file:

```
...
DATA_SOURCES=["MANUAL","YAHOO"]
GHOSTFOLIO_VERSION=1.282.0
...
```

**Reply by rulim34** [reply]
Ok, looks like that's working. But how to use the data source other than Yahoo on market data setting? I've supplied the Alpha Vantage API key, but when I removed Yahoo and tried to use only Alpha Vantage, the result is empty.
